plugins/flytekit-aws-sagemaker/tests/test_boto3_agent.py

```diff
@@ -161,6 +163,7 @@ async def test_agent(mock_boto_call, mock_return_value):
         if "pickle_check" in mock_return_value[0][0]:
             assert "pickle_file" in outputs["result"]
         else:
+            outputs["result"] = msgpack.loads(base64.b64decode(outputs["result"]))
```

Comment:
https://github.com/flyteorg/flytekit/blob/9a9f5b225b8b1ef97f593ae7519d2d7933dbebf1/flytekit/interaction/string_literals.py#L44-L45

Because someone serialize the `bytes` in the binary IDL here before.